TODO.md

```
@@ -6,10 +6,8 @@
 - 🌲 Media: pine
 - 🪵 Server: cedar
-- 🌱 Live USB OS: sapling
-- 🍀 Main VM: clover
-- Install USBs: [hostname]-seed (fern-seed, oak-seed, cedar-seed)
 - VMs: [hostname]-vm (fern-vm, oak-vm, cedar-vm)
 
 - [ ] hydenix: some yubikey touch detection for hyprlock and waybar <https://github.com/maximbaz/yubikey-touch-detector>
-- [ ] hydenix: rofi theme for browser search using unduck shebang
\ No newline at end of file
+- [ ] bluemail
+- [ ] obsi wallpaper
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)